.github/workflows/gardening.yml

```diff
@@ -0,0 +1,53 @@
+name: Gardening
+on:
+  pull_request: # When a PR is opened, edited, updated, or a label is added.
+    types: [opened, synchronize, labeled, edited]
+  push:
+    branches:
+      - master # Every time a PR is merged to master.
+
+jobs:
+  triage:
+    name: "Label PRs"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/labeler@v2
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  # review-crew-afk:
+  #   name: "Review check"
+  #   runs-on: ubuntu-latest
+  #   if: github.event_name == 'pull_request' && github.event.action == 'labeled' && github.event.label.name == '[Status] Needs Review'
+  #   steps:
+  #     - name: Comment
+  #       uses: actions/github-script@0.8.0
+  #       with:
+  #         github-token: ${{ secrets.GITHUB_TOKEN }}
+  #         script: |
+  #           github.issues.createComment({
+  #             issue_number: context.issue.number,
+  #             owner: context.repo.owner,
+  #             repo: context.repo.repo,
+  #             body: 'Howdy! The Jetpack team has disappeared for a few days to a secret island lair to concoct new ways to make Jetpack one hundred billion percent better. As a result, your Pull Request may not be reviewed right away. Do not worry, we will be back next week to look at your work! Thank you for your understanding.'
+  #           })
+  repo-gardening:
+    name: "Manage labels and assignees"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Building Action
+        run: npm install
+        working-directory: ./.github/actions/repo-gardening
+
+      - name: Check if a PR fixes an issue
```

Comment:
If this label describes the action here?

Reply:
Fixed in e26d312c62eb431ae802cc328482466cf6330f99